### ISSUE TITLE
does votenet suit for real-time detection?

### ISSUE THREAD

**Issue by cjhyy7190** [facebookresearch/votenet#18]
the votenet acquire a high accuracy than other net.but I still want to know if votenet can be applied to real-time detection?

**Comment by charlesq34** [comment]
Hi @cjhyy7190 

May I know what is your use case? There might be many definitions of "real-time".

The network takes ~0.1 second to run inference on a scan (after some small optimization like merging conv and batchnorm), so I think it is applicable for real-time applications.

**Comment by cjhyy7190** [comment]
Hi,when I use the votenet to detect my own point clouds.I refer to the demo.py .But when I use the checkpoint.tar which is acquired by training sun rgbd dataset ,the script mentioned me cannot find this file.So I check this file,although it has 10.97MB ,but when I want to open it ,it shows there isn't any files.The demo_files document also has this question.How can I do to solve this problem?Thank you.

**Comment by charlesq34** [comment]
@cjhyy7190 

Have you downloaded the sample point clouds and pre-trained models according to the README?
